src/types/Verification.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * @packageDocumentation
+ * @module IPresentation
+ */
+
+import * as gabi from '@kiltprotocol/portablegabi'
+import CType from '../ctype/CType'
+
+export interface IPresentationReq {
```

Comment:
Let's move this to the file, where it is used

Reply:
fixed in 769fac0dd1e7d9d952e50764df48c5109f380f51